src/Workspaces/Core/Portable/CodeFixesAndRefactorings/FixAllLogger.cs

```diff
@@ -38,21 +39,39 @@ internal static class FixAllLogger
         private const string TotalDiagnosticsToFix = nameof(TotalDiagnosticsToFix);
         private const string TotalFixesToMerge = nameof(TotalFixesToMerge);
 
-        public static void LogState(FixAllState fixAllState, bool isInternalCodeFixProvider)
+        public static void LogState(IFixAllState fixAllState, bool isInternalProvider)
         {
-            Logger.Log(FunctionId.CodeFixes_FixAllOccurrencesContext, KeyValueLogMessage.Create(m =>
+            FunctionId functionId;
+            string providerKey;
+            switch (fixAllState.FixAllKind)
+            {
+                case FixAllKind.CodeFix:
+                    functionId = FunctionId.CodeFixes_FixAllOccurrencesContext;
+                    providerKey = CodeFixProvider;
+                    break;
+
+                case FixAllKind.Refactoring:
+                    functionId = FunctionId.Refactoring_FixAllOccurrencesContext;
```

Comment:
tbh, i'm not sure if we care that much, right?

Reply:
we're already going to say which provider we have... so we already distinguish on what matters.  having a distinct id to bucket fix all for fixes vs refactorings doesn't seem relevant (afaict).

Reply:
Let me sync up with @mikadumont if we care about differentiating between a FixAll for code fixes vs FixAll for refactorings with respect to telemetry. For example, do we want to find out how many FixAll for refactorings were invoked by users over a time frame?